src/scss/settings/global/color/_color.scss

```diff
@@ -82,3 +82,9 @@ $color-orderVerificationRequired:   #e89fae;
 // -----------------------------------------------------------------------------
 
 $color-errorLighter:                #fffafa;
+
+// Mollie
+// -----------------------------------------------------------------------------
+
+$color-mollie:                      #4496f6;
```

Comment:
usually, you should stick to the default checkout colours. These new colours might not work very well with custom themes or theme overrides.